scripts/scripts.js

```diff
@@ -259,7 +259,8 @@ async function fetchWistiaMetadata(videoUrl) {
  */
 function buildHeroBlock(main) {
   // blog pages don't use the hero block
-  if (document.querySelector('body.blog-page')) {
+  // If hero block already added on the page
+  if (document.querySelector('body.blog-page') || document.querySelector('.hero')) {
```

Comment:
May want to be more specific here on this selector, given that none of the sections or block decoration has happened yet.
```suggestion
  if (document.querySelector('body.blog-page') || document.querySelector(':scope > div > div.hero')) {
```